app/main.py

```diff
@@ -2,0 +1,36 @@
+class Animal:
+    alive = []
```

Comment:
The `alive` list is a class variable, meaning it is shared across all instances of `Animal`. If this is not the intended behavior, consider using an instance variable instead.